### ISSUE TITLE
The writing is on the wall. One day,mozilla will switch to blink

### ISSUE THREAD

**Issue by ghost** [BrowserWorks/waterfox#816]
@MrAlex94 

I'm sure you all know now, that microsoft, will use blink for edge.
If even microsoft gave up, that means that's almost certain that mozilla will do the same at some point in the future.

So.... that means that the plan of rebasing to the latest firefox ESR and porting the extensions to that will not hold for very long. The only long term option left, would be to make some sort of wrapper around blink, and port the extensions to that. But that would take years.... if ever done.

In conclusion, waterfox is screwed....

**Comment by hook321** [comment]
This might be better for r/waterfox since it's not something that would likely happen anytime soon, although I doubt Mozilla will end up switching to Blink, given the tone of their blog post. https://blog.mozilla.org/blog/2018/12/06/goodbye-edge/

**Comment by MrAlex94** [comment]
Not a Waterfox issue.

I also sincerely doubt it.

**Comment by ghost** [comment]
You should consider the risk of all your efforts turning to dust....

i was thinking of some kind of wrapper around blink. That would have been a huge engineering project to build making it hopeless....
Then i came across this thing.  
https://github.com/electron/electron

> The Electron framework lets you write cross-platform desktop applications using JavaScript, HTML and CSS. It is based on Node.js and Chromium and is used by the Atom editor and many other apps.

If possible, how difficult would it be to port the extensions to that?
